internal/prompt/prompt.go

```diff
@@ -119,14 +119,14 @@ func RepositoryProjectType() *promptui.Select {
 
 func RepositoryHomepage(value string) Prompt {
 	return Prompt{promptui.Prompt{
-		Label:   "Repository homepage",
+		Label:   "Homepage",
 		Default: value,
 	}}
 }
 
 func RepositoryURL(value string) Prompt {
 	return Prompt{promptui.Prompt{
-		Label:   "Repository url",
+		Label:   "Source code repository url",
```

Comment:
I really like this change, makes it more understandable what this value is.

Reply:
🙏